### PR TITLE
Feature/improve job queue

### PR DESCRIPTION
**Pull request by firmanJS**
## OVERVIEW

modify job queue send an email, send a link for download xlsx not send file because limit on aws ses

### CODE REVIEW

**Review by khihadysucahyo — APPROVED** [review]

File: helpers/email/index.js [hunk 1]
```diff
@@ -20,45 +19,44 @@ smtpTrans.verify(function(error, success) {
   if (error) {
     console.log(error);
```
Comment (sandisunandar99) [inline helpers/email/index.js:20]: maybe this log prefer send to sentry for capture error, please consider this..

Comment (firmanJS) [reply helpers/email/index.js:20]: if sent to sentry, there will be many sentry messages, this is to make it easier to trace in grafana @sandisunandar99

Comment (sandisunandar99) [reply helpers/email/index.js:20]: on grafarana we can't get email notification for error,,

Comment (firmanJS) [reply helpers/email/index.js]: okay i change to sentry

Comment (firmanJS) [reply helpers/email/index.js]: up @sandisunandar99

**Review by sandisunandar99 — APPROVED** [review]
GO..go

**Pull request merged**